build/make/dependencies-glide.mk

```diff
@@ -1,3 +1,5 @@
+##@ Glide dependency management
```

Comment:
Glide is so deprecated, I think we should not include comments here because it would pollute the `make help` output in an unnecessary way. 

:thinking:  Come to think of it, I think we should remove the glide dep makefile in the first place because we don't need it any more, and `go mod` is a future proof way of handling dependencies.

Reply:
I agree on that, but we have to make sure that none of our projects uses it any more before we remove it. Hm, or we remove it and release a new major version :thinking:

Reply:
same counts for go dep. These two can be deleted